src/main/java/com/amihaiemil/zold/RtWallet.java

```diff
@@ -143,10 +144,27 @@ public void pay(
     }
 
     @Override
-    public void find(final String id, final String details) {
-        throw new UnsupportedOperationException(
-            "Not yet implemented. If you can contribute please, do it at "
-            + "https://github.com/amihaiemil/zold-java-client"
+    public JsonArray find(
```

Comment:
@ammaratef45 these changes are not related to origin issue #44, most probably there's another ticket to implement it, if you want to implement this method you can ask ARC to assign you that ticket and submit new PR with these changes.

Reply:
@g4s8 They are in a different PR, I made those changes on the master branch of my fork and had to get them with me as the other PR is not closed

Reply:
@ammaratef45 it looks like this PR doesn't depend on #43, so you can just submit two independent PRs